tests/cli/test_cli_options.py

```diff
@@ -618,6 +620,33 @@ def main(args):
             ):
         return_flag = False
 
+    #
+    # Test --universalindent
+    #
+    if not check_uncrustify_output(
+            uncr_bin,
+            parsed_args,
+            args_arr=['-o', s_path_join(sc_dir, 'results/universalindent.cfg'),
+                      '--universalindent'],
+            gen_expected_path=s_path_join(sc_dir, 'output/universalindent.cfg'),
+            gen_result_path=s_path_join(sc_dir, 'results/universalindent.cfg'),
+            out_result_manip=reg_replace(r'version+', '')
```

Comment:
I think you meant `r'version=.+'` for the pattern? ...and you might want to consider making the replacement `'version='` instead of empty, so we still verify that that field was outputted.